dom.go

```diff
@@ -150,6 +156,11 @@ func (e Node) buildHTML(sb *strings.Builder) *strings.Builder {
 	return sb
 }
 
+func (e Node) isVoidElement() bool {
+	ele := []string{"area", "base", "br", "col", "command", "embed", "hr", "img", "input", "keygen", "link", "meta", "param", "source", "track", "wbr"}
```

Comment:
should i update this to a map instead?

Reply:
updated to map to avoid having range thro the slice

Reply:
As per the other comment, we’re not keeping this function, but if we were to keep it I’d prefer a value like a set of string instead of a function returning bool; more useful